FaultTolerantHdfsAccessor.go

```diff
@@ -43,6 +43,9 @@ func (this *FaultTolerantHdfsAccessor) OpenRead(path string) (ReadSeekCloser, er
 		}
 		if IsSuccessOrBenignError(err) || !op.ShouldRetry("[%s] OpenRead: %s", path, err) {
 			return nil, err
+		}else{
```

Comment:
Actually, I do not think `else` is needed here. If not aborting the retry logic, reset the connection as always, which is also the logic.